lib/components/Snackbar/SnackbarWrapper.tsx

```diff
@@ -0,0 +1,43 @@
+import { StandalonePlaceholderRegularIcon } from "@deriv/quill-icons/Standalone";
+import { Snackbar } from ".";
+import React, { useState } from "react";
+
+export const SnackbarWrapper = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const handleActionClick = () => {
+        console.log("clicked");
+        handleClose();
+    };
+
+    const handleOpen = () => {
+        setIsOpen(true);
+    };
+
+    const handleClose = () => {
+        setTimeout(() => {
+            setIsOpen(false);
+        }, 1000);
+    };
+
+    return (
+        <div>
+            <button onClick={handleOpen} style={{ color: "white" }}>
+                Click me
+            </button>
+            <Snackbar
+                icon={
+                    <StandalonePlaceholderRegularIcon
+                        fill="#ffffff"
+                        iconSize="sm"
+                    />
+                }
+                message="Unable to upload selected photos. \n The app will retry in 5 seconds."
```

Comment:
@henry-deriv What is the use of this component - SnackbarWrapper?  Do we need to pass a default message?

Reply:
im just setting up for next task which is to create a wrapper to keep track of snackbars. But im passing functions into snackbar so that actions inside snackbar itself works. Like closing the snackbar and clicking on action.